tools/extras/install_srilm.sh

```diff
@@ -24,7 +24,7 @@ tar -xvzf ../srilm.tgz
 # set the SRILM variable in the top-level Makefile to this directory.
 cp Makefile tmpf
 
-cat tmpf | awk -v pwd=`pwd` '/SRILM =/{printf("SRILM = %s\n", pwd); next;} {print;}' \
+cat tmpf | gawk -v pwd=`pwd` '/SRILM =/{printf("SRILM = %s\n", pwd); next;} {print;}' \
```

Comment:
Kaldi requires awk to be installed anyway, so using awk is fine and actually recommended/preferred for Kaldi scripts...
we check for gawk because SRILM does not work without it.